commit 4ecc131848c942196fe11f46468574da5cff19f4
Author: Sebastian Ott <sebott@linux.vnet.ibm.com>
Date:   Tue Oct 9 15:57:54 2012 +0200

    udev: path_id - add scm support
    
    Add support for scm block devices. Introduced here:
    http://git.kernel.org/?p=linux/kernel/git/torvalds/linux.git;a=commitdiff;h=f30664e2
    
    Signed-off-by: Sebastian Ott <sebott@linux.vnet.ibm.com>

---
 extras/path_id/path_id.c |    3 +++
 1 file changed, 3 insertions(+)

--- a/extras/path_id/path_id.c
+++ b/extras/path_id/path_id.c
@@ -509,6 +509,9 @@ int main(int argc, char **argv)
 		} else if (strcmp(subsys, "virtio") == 0) {
 			path_prepend(&path, "virtio-pci-%s", udev_device_get_sysname(parent));
 			parent = skip_subsystem(parent, "virtio");
+                } else if (strcmp(subsys, "scm") == 0) {
+                        path_prepend(&path, "scm-%s", udev_device_get_sysname(parent));
+                        parent = skip_subsystem(parent, "scm");
 		}
 
 		parent = udev_device_get_parent(parent);
